cypress/e2e/rich-text/RichTextEditor.spec.ts

```diff
@@ -1832,6 +1842,134 @@ describe('Rich Text Editor', { viewportHeight: 2000 }, () => {
     }
   });
 
+  describe('Embedded Resource Blocks', () => {
```

Comment:
I'm wondering could you please move these tests to a separate file? Something like `RichTextEditor.EmbeddedResourceBlocks.spect.ts`. Cypress can only split tests by files and not individual describes to we are currently extracting out some of these describes to their own files

Reply:
Done